gems/pending/VolumeManager/LVM/scanner.rb

```diff
@@ -86,14 +86,13 @@ def self.labelScan(d)
     pvh
   end # def self.labelScan
 
-  private
-
```

Comment:
Sorry, I think there's overlap with #12041 in this file.  I'm sorry, I didn't realize the two different COPs would point out some of the same code.